### DISCUSSION TITLE
Is there any kind of logging available?

### DISCUSSION THREAD

**Discussion by hash2k2** [HomeITAdmin/nextcloud_geoblocker#59] [Q&A]
Question in the subject.
Is there any kind of logging and where can I find it?

**Comment by HomeITAdmin** [comment]
yes, not allowed login attempts are at least logged. Depending on config they are also blocked. And problems with the configured localication service are also logged during login.

It can be found where all your Nextcloud logging is. That depends on the configuration of your Nextcloud instance. I think the default is logging to a file in <Nextcloud Root>/data/nextcloud.log . You can check e.g. in your config.php.
